### PR TITLE
Re-intruduce REST API version bump

### PR DESCRIPTION
**Pull request by tomassatinsky**
Re-introducing the REST API version bump that will be released as MAJOR version `12.0.0`
---

# PR checklist

- [ ] Change was tested using dev-release in Analytical Designer and Dashboards (if applicable).
- [ ] Change is described in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Migration guide (for major update) is written to [CHANGELOG.md](../blob/master/CHANGELOG.md).

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-oss-check/36/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/461/ : SUCCESS in 1m 03s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-oss-check/37/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/463/ : SUCCESS in 1m 47s

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,9 +8,16 @@ The REST API versions in the table are just for your information as the values a
 
 |gooddata-js version | REST API version
 |:---:|:---:
+|\>= 11.20.0|4
```
Comment (xMort) [inline CHANGELOG.md]: The version `12.0.0` should be here instead.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,9 +8,16 @@ The REST API versions in the table are just for your information as the values a
...
 |\>= 10.0.0|3
 |<= 9.0.1|2
 
+<a name="11.20.0"></a>
```
Comment (xMort) [inline CHANGELOG.md]: The version `12.0.0` here as well.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-merge-compliance-check/107/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/465/ : SUCCESS in 1m 02s

**Pull request merged**